### DISCUSSION TITLE
notes/Cramer-Rao/

### DISCUSSION THREAD

**Discussion by giscus** [x2rs/x2rs.github.io#10] [Giscus Comments]
# notes/Cramer-Rao/

My personal documentation website.

https://x2rs.github.io/notes/Cramer-Rao/

**Comment by x2rs** [comment]
test Cramer-Rao

$$
D(\hat{\theta}) \geq \frac{1}{n E \left[ \left( \frac{\partial}{\partial \theta} \ln p(X, \theta) \right)^2 \right]}
$$
